manager/routes/routes.go

```diff
@@ -46,6 +46,8 @@ func InitAPI(api *gin.RouterGroup, config docs.EndpointsConfig) { // nolint: fun
 	systems.GET("/:inventory_id/packages", controllers.SystemPackagesHandler)
 	systems.DELETE("/:inventory_id", controllers.SystemDeleteHandler)
 
+	api.GET("/system_tags", controllers.SystemTagListHandler)
```

Comment:
Should this be rather `/tags` or is it okay with `/system_tags`. Inventory uses the former.

Reply:
I prefer `/tags` because it's more about "tags used in this account" not tags tied to a particular system.

Reply:
Ok, I'll change it.

Reply:
Done.